src/ducks/Watchlists/Actions/Edit/EditForm/index.module.scss

```diff
@@ -86,3 +87,76 @@
 .description__label {
   margin: 8px 0 4px;
 }
+
+.mrhalf {
+  margin-right: 0.5rem;
```

Comment:
Better to use pixel values, since it's easier to understand and it doesn't depend on some other value